### ISSUE TITLE
No longer works on TorrentLeech after they added cloudflare protection

### ISSUE THREAD

**Issue by newadventure079** [bogenpirat/remote-torrent-adder#355]
I tried to add a torrent from TorrentLeech and RTA's notification comes back with a 403 error.  

Looking at my network sniffer, it seems to be because of cloudflare.  The first part of the html returned is cloudflare's Please Wait page.

`<!DOCTYPE html><html lang="en-US"><head><title>Just a moment...</title>`

**Comment by Andysimcoe** [comment]
Good catch, same happening on hd-torrents, they're using Cloudflare too, though interestingly it does work for me in Firefox but not Chrome or Chromium based browsers.

**Comment by unfussyorg** [comment]
This just started happening to me today, no setup changes made in the extension. It works perfectly on one tracker, but not TL. TL was working without issue the last time I visited.
